templates/rhba70-businesscentral.yaml

```diff
@@ -0,0 +1,326 @@
+  description: 'Custom hostname for http service route.  Leave blank for default hostname,
+    e.g.: <application-name>-rhbacentr-<project>.<default-domain-suffix>'
+  name: BUSINESS_CENTRAL_HOSTNAME_HTTP
+  value: ''
+  required: false
+- displayName: Business Central Custom https Route Hostname
+  description: 'Custom hostname for https service route.  Leave blank for default
+    hostname, e.g.: secure-<application-name>-rhbacentr-<project>.<default-domain-suffix>'
+  name: BUSINESS_CENTRAL_HOSTNAME_HTTPS
+  value: ''
+  required: false
+- displayName: Database JNDI Name
+  description: Database JNDI name used by application to resolve the datasource, e.g. java:/jboss/datasources/ExampleDS
+  name: DB_JNDI
+  value: java:jboss/datasources/ExampleDS
+  required: false
+- displayName: Database Name
+  description: Database name
+  name: DB_DATABASE
+  value: root
+  required: true
+- displayName: Business Central Server Keystore Secret Name
+  description: The name of the secret containing the keystore file
+  name: BUSINESS_CENTRAL_HTTPS_SECRET
+  value: businesscentral-app-secret
+  required: false
+- displayName: Business Central Server Keystore Filename
+  description: The name of the keystore file within the secret
+  name: BUSINESS_CENTRAL_HTTPS_KEYSTORE
+  value: keystore.jks
+  required: false
+- displayName: Business Central Server Certificate Name
+  description: The name associated with the server certificate
+  name: BUSINESS_CENTRAL_HTTPS_NAME
+  value: jboss
+  required: false
+- displayName: Business Central Server Keystore Password
+  description: The password for the keystore and certificate
+  name: BUSINESS_CENTRAL_HTTPS_PASSWORD
+  value: mykeystorepass
+  required: false
+- displayName: Database Username
+  description: Database user name
+  name: DB_USERNAME
+  from: user[a-zA-Z0-9]{3}
+  generate: expression
+  required: true
+- displayName: Database Password
+  description: Database user password
+  name: DB_PASSWORD
+  from: "[a-zA-Z0-9]{8}"
+  generate: expression
+  required: true
+- displayName: Datasource Minimum Pool Size
+  description: Sets xa-pool/min-pool-size for the configured datasource.
+  name: DB_MIN_POOL_SIZE
+  required: false
+- displayName: Datasource Maximum Pool Size
+  description: Sets xa-pool/max-pool-size for the configured datasource.
+  name: DB_MAX_POOL_SIZE
+  required: false
+- displayName: Datasource Transaction Isolation
+  description: Sets transaction-isolation for the configured datasource.
+  name: DB_TX_ISOLATION
+  required: false
+- displayName: ImageStream Namespace
+  description: Namespace in which the ImageStreams for Red Hat Middleware images are
+    installed. These ImageStreams are normally installed in the openshift namespace.
+    You should only need to modify this if you've installed the ImageStreams in a
+    different namespace/project.
+  name: IMAGE_STREAM_NAMESPACE
+  value: openshift
+  required: true
+- displayName: MySQL Image Stream Tag
```

Comment:
This template is only for Business Central and there is no need for a DB so all environment variables for DB can be removed.

Reply:
fixed by taking out the DB section for Business Central